openstack/loadbalancer/v2/monitors/requests.go

```diff
@@ -122,6 +128,10 @@ type CreateOpts struct {
 	// is not specified, it defaults to "GET". Required for HTTP(S) types.
 	HTTPMethod string `json:"http_method,omitempty"`
 
+	// The HTTP version used for requests by the Monitor. If this attribute
+	// is not specified, it defaults to 1.0". Required for HTTP(S) types.
+	HTTPVersion float32 `json:"http_version,omitempty"`
```

Comment:
What a strange type for http_version, I'd personally gone for string... However it's in line with https://github.com/openstack/octavia/blob/master/octavia/api/v2/types/health_monitor.py#L48-L49 and that's all that matters.